magprime/templates/regextra.html

```diff
@@ -77,9 +77,21 @@
         {% endif %}
 
         // MAGFest uses tiered badge pricing based on caps, so we want to link to our FAQ explaining this
-        if ($('.prereg-price-notice').size()) {
-            $('#reg-types').append("<div class='help-block col-sm-9 col-sm-offset-3'>Prices may increase before the set date. {{ macros.popup_link("http://magfest.org/faq#-KOpcFnNhdsUIqvvf6VB", "Why is this?")|e }}</div>");
-        }
+        {% if c.BADGES_LEFT_AT_CURRENT_PRICE %}
+            {% if c.BADGES_LEFT_AT_CURRENT_PRICE == -1 %}
+                {% set badges_left_text = 'Limitless' %}
+            {% elif c.BADGES_LEFT_AT_CURRENT_PRICE <= 100 %}
+                {% set badges_left_text = 'Critical' %}
+            {% elif c.BADGES_LEFT_AT_CURRENT_PRICE <= 250 %}
+                {% set badges_left_text = 'Low' %}
```

Comment:
Maybe "Low" should be "Very Low", and "Medium" should be "Low"? So the tiers end up being:
* Almost Gone
* Very Low
* Low
* High

Just thinking out loud on this one. "Medium" is just kind of a blah word. As an attendee, I'm not even sure what I'd think it meant. I do think it's fair to say that fewer than 500 badges out of 20,000 equates to "Low".

What do you think?